contracts/token/ERC1155/MultiToken.sol

```diff
@@ -89,22 +99,27 @@ contract MultiToken is
         _unpause();
     }
 
-    function setApprovalBurnPermission(address burner, bool approved)
+    function setBurnApproval(address burner, bool approved)
         external
         onlyOwner
         whenBurnableEnabled
         whenNotPaused
     {
         burnApprovals[burner] = approved;
-        emit BurnPermissionApproval(burner, approved);
+        emit BurnApproval(burner, approved);
+    }
+
+    function setMintApproval(address minter, bool approved) external onlyOwner {
```

Comment:
It looks like missing the `whenNotPaused` compared `setBurnApproval`

Reply:
Fixed. thanks!